docs/markdown/Cross-compilation.md

```diff
@@ -131,6 +132,11 @@ cross compilation you can set them with `[langname]_args =
 [args]`. Just remember to specify the args as an array and not as a
 single string (i.e. not as `'-DCROSS=1 -DSOMETHING=3'`).
 
+*Since 0.51.1* The sys_root property should point to the root of the host
+system path (the system that will run the compiled binaries). This is used
+internally by pkg-config to set the PKG_CONFIG_SYSROOT_DIR environment
```

Comment:
This sentence seems muddled:  PKG_CONFIG_SYSROOT_DIR env var is set by meson and used by pkg-config.